zcash_transparent/src/address.rs

```diff
@@ -407,6 +410,14 @@ impl TransparentAddress {
             }
         }
     }
+
+    /// Derives the P2PKH transparent address corresponding to the given pubkey.
+    #[cfg(feature = "transparent-inputs")]
+    pub fn from_pubkey(pubkey: &secp256k1::PublicKey) -> Self {
```

Comment:
Nit: Does this method actually need to be public? The only place I could find `zcash_transparent::keys::pubkey_to_address` being used is in `zcash-devtool inspect`, and the code there is doing manual script parsing that will all get replaced by `zcash_script`. I guess it's not a particularly harmful method, but in most cases what we want is a way to go from some type containing a `PublicKey` to `TransparentAddress`, rather than directly from `PublicKey` (and the deprecation message on `pubkey_to_address` implied this).

Reply:
It's needed later in `zcash_client_sqlite`.